### DISCUSSION TITLE
Persistent TCP connection possible?

### DISCUSSION THREAD

**Discussion by ChPfisterer** [NginxProxyManager/nginx-proxy-manager#3329] [Q&A]
Hey guys, great work on the NGINX Proxy Manager!

Still there is one question, that I cannot answer myself.
Is it possible to establish a persistent TCP connection and send data?
Like: "External" Port 443 -> "NGINX ProxyManager" -> "Container" Port 1234

I am not talking about Websocket (ws://...), but persistent TCP connection (tcp://...).
I thought about NGINX advanced configuration, maybe?

Thanks in advance for your feedback.
Chris

**Comment by olariuromeo** [comment]
I think you can add a custom code that will then be added to each server or main nginx depend of your neds
